ci/docker/Dockerfile.build.ubuntu_gpu_tensorrt

```diff
@@ -18,7 +18,10 @@
 #
 # Dockerfile to run MXNet on Ubuntu 16.04 for CPU
 
-FROM nvidia/cuda:9.0-cudnn7-devel
+FROM nvidia/cuda:10.0-cudnn7-devel
+
+# Avoid interactive package installers.
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Could we put this into the installation script which creates the prompt? Otherwise we'd have to copy and paste the this into all files who referenxe that script

Reply:
At the moment it's specific to cuda10 (and actually the cudnn 7 version referenced here).  It pulls in some new dep that has an interactive installer by default.  I'm not sure why we'd duplicate this env var in a bunch of install scripts, wouldn't it make more sense to put them in our base images to reduce duplication?  I totally don't feel strongly about this though, so whatever file you think this should go in I can move it to.

Reply:
Which package specifically triggers the prompt?

Reply:
I believe the cuda10 version of cudnn includes a package in its closure that triggers the prompt.  I'd have to re-run the Dockerfile without this to see.

Reply:
I'd like to get rid of this if possible. Or at least get it tracked down as closely as possible and then restrict the scope of that variable.

Reply:
So the package that fails is tzdata.  Here's the output:
```
Setting up tzdata (2018i-0ubuntu0.18.04) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Configuring tzdata
------------------

Please select the geographic area in which you live. Subsequent configuration
questions will narrow this down by presenting a list of cities, representing
the time zones in which they are located.

  1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
  2. America     5. Arctic     8. Europe    11. SystemV
  3. Antarctica  6. Asia       9. Indian    12. US
```

Example run: http://jenkins.mxnet-ci.amazon-ml.com/blue/rest/organizations/jenkins/pipelines/mxnet-validation/pipelines/unix-gpu/branches/PR-13874/runs/1/nodes/42/log/?start=0

Reply:
Moved this over to ubuntu core install file. Is that your preference @marcoabreu?

About the scoping, I'm actually not sure why we'd want to set this at a narrow scope.  We don't want interactive installers at any level in the CI right?  I would think you'd prefer disabling this on as broad a level as possible.